src/ui/transforms.js

```diff
@@ -39,41 +43,46 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         });
     };
 
-    /* A transform to turn an array into a delimited string
-     * Values can be optionally be
-     * included only if they are string value and are truthy (i.e. not an empty
-     * string). Values can also be accessed via a specific path relative to each
-     * term.
-     * - separator: the string delimiter to be inserted between each term
-     * - stringOnly (optional): flag to exclude all falsy or non-string values
-     * - path (optional): an EL path on each item in the terms collection
+    sjrk.storyTelling.transforms.stringToArray.invert = function (transformSpec) {
+        transformSpec.type = "sjrk.storyTelling.transforms.arrayToString";
+        return transformSpec;
+    };
+
+    /* A transform to turn an array into a delimited string.
+     * Values can also be accessed via a specific object path relative to each term.
+     * It is partly invertible via "sjrk.storyTelling.transforms.stringToArray".
+     * - "delimiter" (optional): the delimiter to be inserted between each term
+     * - "stringOnly" (optional): flag to allow only non-empty strings
+     * - "path" (optional): an EL path on each item in the terms collection
      */
     fluid.defaults("sjrk.storyTelling.transforms.arrayToString", {
-        "gradeNames": [ "fluid.standardTransformFunction", "fluid.multiInputTransformFunction" ],
-        "inputVariables": {
-            "separator": ", ",
-            "stringOnly": "",
-            "path": ""
-        }
+        gradeNames: ["fluid.standardTransformFunction"],
+        invertConfiguration: "sjrk.storyTelling.transforms.arrayToString.invert"
     });
 
-    sjrk.storyTelling.transforms.arrayToString = function (input, extraInputs) {
-        var contentString = "";
-        var separator = extraInputs.separator();
-        var stringOnly = extraInputs.stringOnly();
-        var path = extraInputs.path();
+    sjrk.storyTelling.transforms.arrayToString = function (input, transformSpec) {
+        var delimiter = transformSpec.delimiter || ", ",
+            stringOnly = transformSpec.stringOnly,
+            path = transformSpec.path || "";
+
+        stringOnly = stringOnly === undefined ? true : stringOnly;
```

Comment:
If you want to also convert `null` to true, you may want to use `fluid.isValue` as the check. Similar for the `trim` option in sjrk.storyTelling.transforms.stringToArray.